src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -2889,7 +2889,7 @@ public enum Deoxys implements LogicCardInfo {
             onAttack {
               damage 80
               delayed {
-                before APPLY_ATTACK_DAMAGES, {
+                after PROCESS_ATTACK_EFFECTS, {
```

Comment:
Interesting. Remembering to verify the order of effects in EffectType.java seems like a good idea when doing this stuff I guess.

Reply:
> Remembering to verify the order of effects in EffectType.java seems like a good idea when doing this stuff I guess.

...Yes. That's exactly what I did~~, and not just follow what worked correctly in other cards blindly. 👀~~